backend/src/db/migrations/20231017141022-fix-event-dates.ts

```diff
@@ -0,0 +1,68 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+Transform event start and end date of format 'YYYY-MM-DD HH:MM:SS' in CEST
+to ISOString in UTC.
+`
+
+export async function up(next) {
+  const driver = getDriver()
+  const session = driver.session()
+  const transaction = session.beginTransaction()
+
+  try {
+    const events = await transaction.run(`
```

Comment:
To make this universal we could to the following:

```
if (post.createdAt is before this migration date) {// we specify a date here
   // We assume German time as timezone
   if(post.eventStart is German Summer Time){
      post.eventStart -=2h
      post.eventEnd -= 2h
   } else {
      post.eventStart -=1h
      ...
   }
}
```

You can use https://stackoverflow.com/questions/11887934/how-to-check-if-dst-daylight-saving-time-is-in-effect-and-if-so-the-offset to determine if DST is in effect.

Reply:
Really needed? I think not